pkg/apiserver/diagnose/report.go

```diff
@@ -324,7 +324,7 @@ func GetDiagnoseReport(startTime, endTime string, db *gorm.DB) (TableDef, error)
 	table := TableDef{
 		Category:  []string{CategoryDiagnose},
 		Title:     "diagnose",
-		CommentEN: "Automatically diagnose the cluster problem and record the problem in below table.",
+		CommentEN: "diagnose_comment",
 		CommentCN: "",
```

Comment:
Maybe we can combine the "CommentEN" and "CommentCN" to "Comment".

Reply:
+1. We should merge them into one `Comment`.